app/src/main/kotlin/com/metrolist/music/utils/DiscordRPC.kt

```
@@ -35,19 +40,22 @@ class DiscordRPC(
+        Timber.d("updateSong: title=\"%s\", artist=\"%s\", activityType=%s",
+            song.song.title, song.artists.joinToString { it.name }, activityType)
         val currentTime = System.currentTimeMillis()
 
-        val adjustedPlaybackTime = (currentPlaybackTimeMillis / playbackSpeed).toLong()
+        val adjustedPlaybackTime = (currentPlaybackTimeMillis / validPlaybackSpeed).toLong()
         val calculatedStartTime = currentTime - adjustedPlaybackTime
 
-        val songTitleWithRate = if (playbackSpeed != 1.0f) {
-            "${song.song.title} [${String.format("%.2fx", playbackSpeed)}]"
+        val songTitleWithRate = if (validPlaybackSpeed != 1.0f) {
+            "${song.song.title} [${String.format("%.2fx", validPlaybackSpeed)}]"
```

detekt (1.23.8)

[warning] String.format("%.2fx", validPlaybackSpeed) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)